mesheryctl/internal/cli/root/system/testdata/reset/reset_kubernetes.output.golden

```diff
@@ -2,9 +2,9 @@ Meshery resetting...
 
 Current Context: local
 Channel: stable
-Version: v0.5.56
+Version: v0.6.6
 Platform: kubernetes
 
 Fetching Meshery Server and Meshery Operator manifests for  local context...
-downloading manifest files from https://github.com/meshery/meshery/tree/v0.5.56/install/deployment_yamls/k8s
-...meshconfig has been reset to default settings.
+downloading manifest files from https://github.com/meshery/meshery/tree/v0.6.6/install/deployment_yamls/k8s
+...meshconfig has been reset to default settings.
```

Comment:
Unit tests are probably failing because you removed the final newline `\n` from this file.

Reply:
Great catch, I didn't even notice that, I thought I copied the file'c ontent with the newline. I updated my PR